File: TaskTracker.Model/UserSpace/UserSpaceModel.cs
using TaskTracker.Model.TaskList;
using TaskTracker.Model.TaskStatusGroup;
using TaskTracker.Model.User;

namespace TaskTracker.Model.UserSpace
{
    public class UserSpaceModel : IModel
    {
        public long Id { get; set; }
        public string Title { get; set; }
        public TaskStatusGroupModel StatusGroup { get; set; }
        public UserModel Owner { get; set; }
        public IEnumerable<TaskListModel>? Lists { get; set; }
    }
}

File: task-tracker-ui/src/app/model/user-space/UserSpaceModel.ts
import TaskListModel from "../task-list/TaskListModel";
import TaskStatusGroupModel from "../task-status-group/TaskStatusGroupModel";
import UserModel from "../user/UserModel";

export default interface UserSpaceModel {
    id: number,
    title: string,
    statusGroup: TaskStatusGroupModel,
    lists: TaskListModel[],
    owner: UserModel
}